### DISCUSSION TITLE
Welcome to leelaa-admin Discussions!

### DISCUSSION THREAD

**Discussion by admin8756** [admin8756/leelaa-admin#11] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 784087189** [comment]
完美，如果有安卓客户端就好了，已经秒杀现在Audiobookshelf
